views/templates/auth/form.tpl

```diff
@@ -0,0 +1,24 @@
+<?php
+$authType = get_data('authType');
+$allowedTypes = get_data('allowedTypes');
+?>
+
+<div class="auth-selector-component">
+    <div class="auth-type-container">
+        <label for="taoPlatformAuthType" class="form_desc"><?=__('Auth type')?></label>
+        <select name="<?= \tao_helpers_Uri::encode(\oat\taoPublishing\model\PlatformService::PROPERTY_AUTH_TYPE)?>" id="taoPlatformAuthType" class="auth-type-selector">
+            <?php foreach($allowedTypes as $type) :?>
+                <option value="<?= $type->getAuthClass()->getUri() ?>"
+                        <?php if ($type->getAuthClass()->getUri() == $authType->getAuthClass()->getUri()) :?>selected="selected"<?php endif; ?>
+                ><?=$type->getAuthClass()->getLabel()?></option>
+            <?php endforeach; ?>
+        </select>
+    </div>
+    <div class="authenticator-settings">
+        <?php foreach ($allowedTypes as $allowedType) :?>
+            <div<?php if ($allowedType->getAuthClass()->getUri() != $authType->getAuthClass()->getUri()) :?> style="display:none"<?php endif; ?>>
```

Comment:
Why don't you use the class `hidden` instead?

Reply:
who knows? :) will change it